requirements.txt

```diff
@@ -1,6 +1,6 @@
 wheel
 setuptools >=61,!=67.2.0
-numpy >=1.16,<1.22
+numpy >=1.16
```

Comment:
Fixed in c5a4089.

Reply:
I checked at this wasn't changed in your commit. I assum that this is because the new `continous-integration.yml` doesn't use `requirements.txt`. Does this mean that `requirements.txt` should simply be removed entirely ?

Reply:
Sorry, I was too fast and did not realize that this was file `requirements.txt` :sweat_smile:

The purpose of `requirements.txt` is to install the correct dependencies for the `h5py` library, which must be installed from the sources with MPI support. `continous-integration.yml` does use `requirements.txt` as you can see [here](https://github.com/pyccel/psydac/blob/7a4e09544e77e5683dc7083989c007fd904cdd7b/.github/workflows/continuous-integration.yml#L91-L96).

Reply:
Really fixed in 3f85376.